go.mod

```diff
@@ -16,7 +16,7 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/klauspost/compress v1.15.0
 	github.com/klauspost/cpuid/v2 v2.0.11
-	github.com/lucas-clemente/quic-go v0.25.0
+	github.com/lucas-clemente/quic-go v0.25.1-0.20220321092029-b7e93b54c9cc
```

Comment:
Do you guys want a release? Just let me know, I can cut one.

Reply:
Let's do one after https://github.com/lucas-clemente/quic-go/issues/3350 :D

Reply:
Can we update this now that #3352 was merged? :eyes:

Reply:
I think you meant https://github.com/lucas-clemente/quic-go/issues/3352, but yep, I'm sure @renbou will be on it as soon as he can!

Reply:
Yep, my bad - thanks!

Reply:
Cutting the release will be my responsibility - but please give CI enough time to run before asking for the release ;)
This PR was literally merge less than 10 minutes ago.

Reply:
All good! Didn't want to sound impatient.

Reply:
v0.26.0 was shipped: https://github.com/lucas-clemente/quic-go/releases/tag/v0.26.0

Reply:
Thank you!